package.json

```diff
@@ -3,18 +3,18 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "start": "serve -s ./build",
+    "start": "react-scripts start",
     "dev": "npm run lint && react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --watchAll",
     "test:coverage": "react-scripts test --watchAll \"-- --coverage\"",
     "test:CI": "CI=true react-scripts test \"-- --coverage\"",
     "coveralls": "cat coverage/lcov.info | coveralls",
     "eject": "react-scripts eject",
-    "lint": "./node_modules/.bin/eslint . --fix"
+    "lint": "./node_modules/.bin/eslint . --fix",
+    "cover": "CI=true react-scripts test \"-- --coverage\""
```

Comment:
@luc-tuyishime, this is repetition as we already have a script for that: `"test:coverage": "react-scripts test --watchAll \"-- --coverage\""`

Reply:
Removed already